pcstac/pcstac/main.py

```diff
@@ -50,6 +55,15 @@
 
 app_settings = get_settings()
 
+items_get_request_model = PCItemCollectionUri
```

Comment:
we can now directly define the `GET - /items` model. This wasn't used previously